### ISSUE TITLE
Add .keycap class to conflict dialog

### ISSUE THREAD

**Issue by cassidyjames** [elementary/settings-keyboard#144]
I don't think it's necessary in the listing (it would probably just be super noisy?), but we should probably use the `.keycap` class to key names in the conflict dialog. It can be confusing when a user has `+` as one of the keys, and since the stylesheet has support for it, I feel like we should use it.

![screenshot](https://user-images.githubusercontent.com/16517095/36750294-7c090734-1bfd-11e8-9779-fa11d75f4f66.png)

**Comment by danrabbit** [comment]
Yeah I was thinking about that. One downside is we'd have to build the dialog manually instead of using Granite.MessageDialog since the primary text is just a single Gtk.Label. Another potential problem though is translations. I'm not sure if Pango accepts the `style` attribute or not

**Comment by lenemter** [comment]
This dialog was removed in https://github.com/elementary/settings-keyboard/pull/539

**Comment by danirabbit** [comment]
I think this is still valid. There's still a dialog

![Image](https://github.com/user-attachments/assets/ffba17da-5b0c-4847-8bca-145739d3534e)
